tests/python/opinfo/test_direct_ops.py

```diff
@@ -218,5 +220,10 @@ def _regex_escape_parenthesis(a: str) -> str:
 def test_errors(op: OpInfo, dtype: torch.dtype):
     for sample, exception_type, exception_regex in op.error_input_generator(op, dtype):
         # TODO skip regex check because direct bindings do not have same error message as frontend.
-        with pytest.raises(exception_type):
+        # Capture error strings to avoid false positives in the CI
+        stdout_capture = io.StringIO()
+        stderr_capture = io.StringIO()
+        with pytest.raises(exception_type), redirect_stdout(
+            stdout_capture
+        ), redirect_stderr(stderr_capture):
```

Comment:
Interesting. So this is only to work with our CI tools for automatic analysis?

Reply:
curious about the updated behavior.

We get an error that doesn't match the expected exception, the updated error log would just give us the real error instead of the mismatched error string in that case?

Reply:
> We get an error that doesn't match the expected exception, the updated error log would just give us the real error instead of the mismatched error string in that case?

For direct bindings, we are only checking the error type for now.
We can get the exception string through pytest:
```
    with pytest.raises(ValueError) as exc_info:
        validate_age(-5)
    assert "Age cannot be negative" in str(exc_info.value)
    assert exc_info.type is ValueError
```
For some reason, python and `stderr` get the same error message twice.